src/ophyd_async/core/_device_filler.py

```diff
@@ -137,6 +137,12 @@ def check_created(self):
                 f"{self._device.name}: {uncreated} have not been created yet"
             )
 
+    @staticmethod
+    def check_device_annotation(annotation: Any) -> None:
+        if not isinstance(annotation, DeviceAnnotation):
+            msg = f"Annotation {annotation} is not a DeviceAnnotation"
+            raise TypeError(msg)
```

Comment:
As this is only used in this file, please could you make it a free private function in this file...
```
def _check_device_annotation(annotation: Any) -> DeviceAnnotation:
```

Reply:
done 8af924f9a25c4a8673e2eeafcfe51786dddba9c2

Reply:
done cc79c3a4598d4ae06df639906742627ff5cc4ef3